src/commands.jl

```diff
@@ -200,19 +231,23 @@ function execute_command(state::DebuggerState, ::Val{:?}, cmd::AbstractString)
     display(
             @md_str """
     Basic Commands:\\
-    - `n` steps to the next line\\
-    - `s` steps into the next call\\
-    - `finish` runs to the end of the function\\
-    - `bt` shows a simple backtrace\\
-    - ``` `stuff ``` runs `stuff` in the current frame's context\\
-    - `fr v` will show all variables in the current frame\\
-    - `f n` where `n` is an integer, will go to the `n`-th frame\\
-    - `q` quits the debugger, returning `nothing`\\
+    - `n`: step to the next line\\
+    - `s`: step into the next call\\
+    - `finish`: run to the end of the function\\
+    - `bt`: show a simple backtrace\\
+    - ``` `stuff ```: run `stuff` in the current function's context\\
+    - `fr [v]`: show all variables in the current frame, `v` defaults to 1\\
+    - `w`\\
+        - `w a expr`: add an expression to the watch list\\
+        - `w`: show all watch expressions evaluated in the current function's context\\
+        - `w c [i::Int]`: clear all or the `i`:th watch expression\\
```

Comment:
Might be better if those commands were consistent with e.g. the Pkg REPL mode, so `w add` for adding watchers and `w rm [i]` or `w remove [i]` for removing them.

Reply:
Good idea